fuel-vm/src/tests/blockchain.rs

```diff
@@ -545,9 +545,9 @@ fn ldc_reason_helper(cmd: Vec<Instruction>, expected_reason: PanicReason, should
     } = receipts.get(0).expect("No receipt")
     {
         assert_eq!(
-            &expected_reason,
+            &Some(expected_reason),
```

Comment:
Makes me wonder if the panic receipt should really just have a non-optional panic reason, instead of instruction result. Seems strange to have the panic reason optional on a panic receipt.